src/apb/engine.py

```diff
@@ -129,31 +130,24 @@ def generate_playbook_files(playbooks_path, roles_path, bindable, skip, apb_name
     ENV = Environment(loader=FileSystemLoader(DAT_PATH))
     template = ENV.get_template(EX_PLAYBOOK_FILE)
 
-    print("Generating playbook files")
-
-    if 'provision' not in skip:
-        # Create provision playbook
-        write_playbook_file(playbooks_path, PROVISION_PLAYBOOK, template, apb_name, 'provision')
-        if 'roles' not in skip:
-            os.mkdir(os.path.join(roles_path, 'provision-' + apb_name))
+    playbooks = ['provision', 'deprovision', 'bind', 'unbind']
 
-    if 'deprovision' not in skip:
-        # Create deprovision playbook
-        write_playbook_file(playbooks_path, DEPROVISION_PLAYBOOK, template, apb_name, 'deprovision')
-        if 'roles' not in skip:
-            os.mkdir(os.path.join(roles_path, 'deprovision-' + apb_name))
+    playbooks_dict = {
+            'provision': PROVISION_PLAYBOOK,
+            'deprovision': DEPROVISION_PLAYBOOK,
+            'bind': BIND_PLAYBOOK,
+            'unbind': UNBIND_PLAYBOOK
+    }
 
-    if 'bind' not in skip:
-        # Create Bind playbook
-        write_playbook_file(playbooks_path, BIND_PLAYBOOK, template, apb_name, 'bind')
-        if 'roles' not in skip:
-            os.mkdir(os.path.join(roles_path, 'bind-' + apb_name))
+    print("Generating playbook files")
 
-    if 'unbind' not in skip:
-        # Create Unbind playbook
-        write_playbook_file(playbooks_path, UNBIND_PLAYBOOK, template, apb_name, 'unbind')
-        if 'roles' not in skip:
-            os.mkdir(os.path.join(roles_path, 'unbind-' + apb_name))
+    for playbook in playbooks:
```

Comment:
You can remove the ```playbook``` list and do ```for playbook in playbooks_dict.keys():```.  That will loop through the dict keys and will get the same functionality as using the list.